src/tech/support-and-on-call.md

```diff
@@ -273,7 +273,7 @@ the `Default API` (not the `ops_team api`) and add them as a responder.
 If you are alerted out of hours you can claim Time Off in Lieu (TOIL) in
 BreatheHR.
 
-* For alerts while you are awake you can claim 0.25 days of TOIL.
+* For alerts while you are awake you can claim 0.25 days* of TOIL.
```

Comment:
I wonder if we just say it onthese lines. 
So  
For alerts while you are awake you can claim 0.25 days/1.75 hours of TOIL

Reply:
Yeah, I wondered that - happy to change it to that. Would you do that throughout the section or just on the first  mention?

Reply:
I think just in this bit. so for 0.25 and 0.5 so we at least let people know and then they can do thier own maths for the next few bits.